bin/node/src/chain_spec.rs

```diff
@@ -282,23 +316,29 @@ fn genesis(
     // NOTE: some combinations of bootstrap chain arguments can potentially
     // lead to duplicated rich accounts, e.g. if a sudo account is also an authority
     // which is why we remove the duplicates if any here
+    // endow as well stash accounts, if any
```

Comment:
this last sentence is just unnecessary

Reply:
My goal here was that since we've added line just below
```rust
     .chain(stakers.iter().cloned())
```
Then it supposed to mean that for some scenarios (for devnet only) `stakers` is not empty and we endow some more account, but I agree the comment might sound  misleading